DFC.ServiceTaxonomy.GraphSync/Activities/AuditSyncIssuesTask.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DFC.ServiceTaxonomy.GraphSync.GraphSyncers;
+using DFC.ServiceTaxonomy.GraphSync.GraphSyncers.Interfaces;
+using DFC.ServiceTaxonomy.Neo4j.Services;
+using Microsoft.Extensions.Localization;
+using Neo4j.Driver;
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.Metadata;
+using OrchardCore.ContentManagement.Metadata.Models;
+using OrchardCore.ContentManagement.Records;
+using OrchardCore.Workflows.Abstractions.Models;
+using OrchardCore.Workflows.Activities;
+using OrchardCore.Workflows.Models;
+using YesSql;
+using YesSql.Services;
+
+namespace DFC.ServiceTaxonomy.GraphSync.Activities
+{
+    public class AuditSyncIssuesTask : TaskActivity
+    {
+        public AuditSyncIssuesTask(
+            ISession session,
+            IGraphDatabase graphDatabase,
+            IUpsertGraphSyncer upsertGraphSyncer,
+            IStringLocalizer<AuditSyncIssuesTask> localizer,
+            IContentDefinitionManager contentDefinitionManager,
+            IEnumerable<IContentPartGraphSyncer> partSyncers)
+        {
+            _session = session;
+            _graphDatabase = graphDatabase;
+            _upsertGraphSyncer = upsertGraphSyncer;
+            T = localizer;
+            _partSyncers = partSyncers.ToDictionary(x => x.PartName ?? "Eponymous");
+            _syncFailedContentItems = new List<ContentItem>();
+            _contentTypes = contentDefinitionManager
+                .ListTypeDefinitions()
+                .Where(x => x.Parts.Any(p => p.Name == "GraphSyncPart"))
+                .ToDictionary(x => x.Name);
+        }
+
+        private readonly ISession _session;
+        private readonly IGraphDatabase _graphDatabase;
+        private readonly Dictionary<string, IContentPartGraphSyncer> _partSyncers;
+        private readonly IUpsertGraphSyncer _upsertGraphSyncer;
+        private IStringLocalizer T { get; }
+
+        public override string Name => nameof(AuditSyncIssuesTask);
+        public override LocalizedString DisplayText => T["Identify sync issues and retry"];
+        public override LocalizedString Category => T["National Careers Service"];
```

Comment:
The category for our (generic) graph sync components has been changed to "Graph", we should have this match.